### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by gwarf**
Various small cosmetic changes on the OneProvider documentation.

### CODE REVIEW

File: content/en/providers/datahub/oneprovider/_index.md [hunk 1]
```diff
@@ -9,19 +9,19 @@ weight: 20
 ---
 
 This documentation covers how to install and configure a OneData OneProvider in
-order to join a new or existing DataHub space. In particular 2 types of
+order to join a new or existing EGI DataHub space. In particular two types of
 installations are available, depending if the provider wants to support the
-space with an an empty storage or if existing data are willing to be exposed via
-the OneProvider.
+space with an empty storage or if existing data should be exposed via the
+OneProvider.
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:15]:
```suggestion
Oneprovider.
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 2]
```diff
@@ -9,19 +9,19 @@ weight: 20
...
 
 ## Requirements for production installation
 
-- Oneprovider
+- OneProvider
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:19]:
```suggestion
- Oneprovider
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 3]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
 
 ## Installation and attach empty storage to the EGI DataHub
 
-The installation of a new OneProvider is performed using the an installation
-script used `onedatify` which will setup the components using Docker and
-Docker-compose
+The installation of a new OneProvider is performed using the `onedatify`
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:35]:
```suggestion
The installation of a new Oneprovider is performed using the `onedatify`
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 4]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
...
+you will have to copy the command already configured with the correct parameters
+for the OneZone to use (`datahub.egi.eu`) and the space to join.
 
 ![image](onedatify-oneprovider.png)
 
 ### Run the command on the host
 
-Paste the copied command in the terminal on the OneProvider machine (as
-superuser).
+Paste the copied command in the terminal on the OneProvider machine as
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:57]:
```suggestion
Paste the copied command in the terminal on the Oneprovider machine as
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 5]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
...
 
-`onedatify logs`
+```shell
+onedatify logs
+```
 
-After the deployment is complete, the following message will be shown, with
-connection details for administration panel for the Oneprovider instance:
+After the deployment is complete, a message will be shown, including connection
+details for the administration panel of the OneProvider:
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:76]:
```suggestion
details for the administration panel of the Oneprovider:
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 6]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
...
 
 ![image](onedatify_step_5.png)
 
-The WebUI Admin portal at port 9443 can be used to administer the OneProvider
-installation.
+This administration panel at port `9443` can be used to administer the
+OneProvider.
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:81]:
```suggestion
Oneprovider.
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 7]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
...
+The installation of a new OneProvider to expose existing datasets to an EGI
 DataHub space is similar to the installation with an empty storage.
 
 When adding support to an existing or new space you should select from the EGI
-DataHub GUI the tab : `Expose Existing dat set` and there you will have just to
-copy the command having already the correct parameters for the OneZone to use
-(datahub.egi.eu) and the space to join.
+DataHub user interface the tab : `Expose Existing dataset` and there you will
+have to copy the command already configured with the correct parameters for the
+OneZone (`datahub.egi.eu`) and the space to join.
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:91]:
```suggestion
Onezone (`datahub.egi.eu`) and the space to join.
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 8]
```diff
@@ -32,74 +32,75 @@ the OneProvider.
...
 
 ![image](onedatify-oneprovider-expose.png)
 
 ### Run the command on the host
 
-Paste the copied command in the terminal on the OneProvider machine (as
-superuser) and follow the instructions as for the case of an empty storage.
+Paste the copied command in the terminal on the OneProvider machine as
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:97]:
```suggestion
Paste the copied command in the terminal on the Oneprovider machine as
```

**Comment by andrea-manzi** [comment]
thanks  a lot! i just checked the correct way to write Oneprovider and Onezone  from the docs (https://onedata.org/#/home/api/stable/oneprovider) so i changed it accordingly

**Review by enolfc — APPROVED** [review]
LGTM!

**Pull request merged**